contracts/0.8.25/vaults/ValidatorConsolidationRequests.sol

```diff
@@ -80,68 +65,102 @@ contract ValidatorConsolidationRequests {
      *      | ----- public key (48 bytes) ----- || ----- public key (48 bytes) ----- | ...
      *
      * @param _refundRecipient The address to refund the excess consolidation fee to.
-     * @param _stakingVault The address of the staking vault contract.
+     * @param _dashboard The address of the dashboard contract.
      * @param _adjustmentIncrease The sum of the balances of the source validators to increase the rewards adjustment by.
      */
-    function addConsolidationRequests(
+    function addConsolidationRequestsAndIncreaseRewardsAdjustment(
         bytes[] calldata _sourcePubkeys,
         bytes[] calldata _targetPubkeys,
         address _refundRecipient,
-        address _stakingVault,
+        address _dashboard,
         uint256 _adjustmentIncrease
-    ) external payable onlyDelegateCall {
+    ) external payable {
         if (msg.value == 0) revert ZeroArgument("msg.value");
         if (_sourcePubkeys.length == 0) revert ZeroArgument("sourcePubkeys");
         if (_targetPubkeys.length == 0) revert ZeroArgument("targetPubkeys");
-        if (_stakingVault == address(0)) revert ZeroArgument("stakingVault");
-
+        if (_dashboard == address(0)) revert ZeroArgument("dashboard");
+        if (_sourcePubkeys.length != _targetPubkeys.length) revert MismatchingSourceAndTargetPubkeysCount(_sourcePubkeys.length, _targetPubkeys.length);
+        
         // If the refund recipient is not set, use the sender as the refund recipient
         if (_refundRecipient == address(0)) {
             _refundRecipient = msg.sender;
         }
 
-        if (_sourcePubkeys.length != _targetPubkeys.length) {
-            revert MismatchingSourceAndTargetPubkeysCount(_sourcePubkeys.length, _targetPubkeys.length);
-        }
-
         VaultHub vaultHub = VaultHub(payable(LIDO_LOCATOR.vaultHub()));
-        VaultHub.VaultConnection memory vaultConnection = vaultHub.vaultConnection(_stakingVault);
-
-        if(vaultConnection.vaultIndex == 0 || vaultHub.isPendingDisconnect(_stakingVault)) {
+        address stakingVault = address(Dashboard(payable(_dashboard)).stakingVault());
```

Comment:
should check that Dashboard owns the vault

Reply:
should say that this contract is strictly for a node operator :warning: role, etc.